docs/apis/batch/libs/gelly.md

```diff
@@ -2051,6 +2052,26 @@ The algorithm takes a directed, vertex (and possibly edge) attributed graph as i
 vertex represents a group of vertices and each edge represents a group of edges from the input graph. Furthermore, each
 vertex and edge in the output graph stores the common group value and the number of represented elements.
 
+### Jaccard Index
+
+#### Overview
+The Jaccard Index measures the similarity between vertex neighborhoods. Scores range from 0.0 (no common neighbors) to
+1.0 (all neighbors are common).
+
+#### Details
+Counting common neighbors for pairs of vertices is equivalent to counting the two-paths consisting of two edges
+connecting the two vertices to the common neighbor. The number of distinct neighbors for pairs of vertices is computed
+by storing the sum of degrees of the vertex pair and subtracting the count of common neighbors, which are double-counted
+in the sum of degrees.
+
+The algorithm first annotates each edge with the endpoint degree. Grouping on the midpoint vertex, each pair of
```

Comment:
Source, target or both endpoints? What is a "midpoint" vertex? Is this standard terminology? It might help to give a small example here.